src/Stratis.Bitcoin.Features.PoA/PoAConsensusOptions.cs

```diff
@@ -91,10 +111,11 @@ public class PoAConsensusOptions : ConsensusOptions
         public int PollExpiryBlocks { get; set; }
 
         /// <summary>
-        /// Defines when V2 of the contract serializer will be used.
-        /// I.e if tip <= ContractSerializerV2ActivationHeight, V1 will be used.
+        /// Leaving this for now for use by Stratis.SmartContracts.CLR.
```

Comment:
The idea is to reduce the chances of possible version incompatibilities where projects may still be using the older `Stratis.SmartContracs.CLR` such as `Stratis.SmartContracts` within say test projects that use network classes that don't have the field available. Can't remember the exact circumstance was that caused issued for me but I concluded this would do more good than harm.